docs/release-notes.md

```diff
@@ -15,6 +15,7 @@ key](https://getfedora.org/security/).
 ### Features
 
 - Add `-c`/`--check` option to check config without producing output
+- Warn if config attempts to reuse partition by label _(fcos 1.6.0-exp)_
```

Comment:
"Warn when a non root disk attempts to reuse partition by label _\_(fcos 1.6.0-exp)\__" WDYT?

Reply:
Yeah i like that, it it too specific if we said "Unless wipe_table is true warn when a non root disk attempts to reuse partition by label _(fcos 1.6.0-exp)_"